src/Rokt-Kit.js

```diff
@@ -47,7 +46,6 @@ var constructor = function () {
                 ) {
                     window.Rokt.createLauncher({
                         accountId: accountId,
-                        sandbox: sandboxMode,
```

Comment:
I think you can replace this line with `sandbox: mParticle.getEnvironment() === 'development',` and continue to use sandbox mode.